apply_extra.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
```

Comment:
This is all commented-out now. If it's on purpose, please remove it.

Reply:
It has been commented out, might become useful again in the future so I shall leave it there